RELEASE-NOTES.txt

```diff
@@ -22,12 +23,11 @@ Pre-rolled CPU models in SimEng include:
 Current limitations (to be addressed in a future release):
 * Dynamically linked binaries are not yet supported in SimEng but developments are currently being undertaken
 * We don't currently support running MPI programs (we’ve had a statically linked OpenMPI single rank program run correctly, but this was painful to build)
-* We only support a subset of Linux's system calls, and these run under emulation. The subset of 42 calls is enough to run all the codes we've tried so far
+* We only support a subset of Linux's system calls, and these run under emulation. The subset of 43 calls is enough to run all the codes we've tried so far
 * Only partial support for micro-oping, mainly for Load and Store operations
-* Only partial support for the ISA - we’ve focused on implementing the instructions we’ve seen generated by the compiler for all the test cases we’ve considered, about 930 different instructions so far (~15% of the entire ISA). You’re likely to come across “instruction not implemented” errors, just let us know when you see these
+* Only partial support for the ISA - we’ve focused on implementing the instructions we’ve seen generated by the compiler for all the test cases we’ve considered, about 1000 different AArch64 instructions (~15% of the entire ISA) and about 150 different RISC-V instructions so far (~33% of the rv64ima target). You’re likely to come across “instruction not implemented” errors, just let us know when you see these
```

Comment:
Where did the 33% RISC-V number come from?

Reply:
Forgot to change this, was meant to say (~33% of the RISC-V ISA supported by Capstone). That okay?